mycroft/util/__init__.py

```diff
@@ -364,3 +366,11 @@ def wait_while_speaking():
     time.sleep(0.1)  # Wait briefly in for any queued speech to begin
     while is_speaking():
         time.sleep(0.1)
+
+
+def stop_speaking():
+    # TODO: Less hacky approach to this once Audio Manager is implemented
+    # Skills should only be able to stop speech they've initiated
+    config = mycroft.configuration.ConfigurationManager.instance()
+    kill([config.get('tts').get('module')])
+    kill(["aplay"])
```

Comment:
Also I this will only stop the current sentence. On Picroft this isn't an issue but on PC it get's a bit weird during pairing.